src/gs-details-page.c

```diff
@@ -2265,6 +2265,16 @@ gs_details_page_network_available_notify_cb (GsPluginLoader *plugin_loader,
 	gs_details_page_refresh_reviews (self);
 }
 
+static void
+gs_details_page_key_pressed_cb (GtkWidget *widget, GdkEventKey *key, GsDetailsPage *self)
+{
+	g_return_if_fail (key != NULL);
+
+	if (key->keyval == GDK_KEY_Escape) {
```

Comment:
I believe gnome-software uses no curly braces for single-line condition blocks.

Reply:
No hard rule AFAIK. But yeah, let's try to keep it simpler without the brackets.